akka-docs/src/main/paradox/testing.md

```diff
@@ -383,6 +383,13 @@ Scala
 Java
 :   @@snip [TestKitDocTest.java]($code$/java/jdocs/testkit/TestKitDocTest.java) { #test-special-probe }
 
+@@@ note
+
+The scala code uses `=> true` in the partial function. But that means nothing special in this context. 
+The `expectMsgPF` checks for the matching and not checks what is the return value.
+
+@@@
```

Comment:
Hmm, I'm confused why we need to explain this -- as in, why would someone care about the `=>` (call by name) being used there?

Reply:
In the scala code: `case Update(id, _) if id == x ⇒ true` this line is calling for refactor to this: `case Update(id, _)   ⇒ id == x` but they make really different things. And it's really hard to understand for the first time seeing only the given examples. So my first idea was to make a note but maybe rewriteing that line to `case Update(id, _) if id == x ⇒ ` or `case Update(id, _) if id == x ⇒ ()` would be even better.